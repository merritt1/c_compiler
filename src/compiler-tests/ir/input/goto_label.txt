void main(void) {
	int i;
	myLabel:
		i = 3;
	goto myLabel;
}