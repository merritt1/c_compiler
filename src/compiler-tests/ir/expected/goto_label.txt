(procBegin,  main)
(label   ,  _UserLabel_main_myLabel)
(addressOf,     r0,      i)
(constInt,     r1,          3)
(storeWord,     r0,      r1)
(goto    ,  _UserLabel_main_myLabel)
(procEnd ,  main)
